package.json

```diff
@@ -9,7 +9,7 @@
     "setup": "yarn install && yarn allow-scripts",
     "lint:eslint": "yarn eslint . --ext ts,js",
     "lint:config-validation": "node ./scripts/validate-configs.js",
-    "lint:misc": "prettier '**/*.json' '**/*.md' '!**/CHANGELOG.md' '**/*.yml' --ignore-path .gitignore",
+    "lint:misc": "prettier '**/*.json' '!**/rules-snapshot.json' '**/*.md' '!**/CHANGELOG.md' '**/*.yml' --ignore-path .gitignore",
```

Comment:
Prettier dislikes something about the additions to the rule snapshot, and I don't think we care if it's linted.

Reply:
Should we start a `.prettierignore` file and place this there instead? I'm worried that this would be easy to miss.

Reply:
Is this a general suggestion, or for this case only? That'd have some benefits, but it'd come at the cost of no longer lazily using the `.gitignore` instead.

Reply:
Ah, okay. Never mind then!